sktime/classification/deep_learning/hf_momentfm_classifier.py

```diff
@@ -0,0 +1,597 @@
+"""Interface for the momentfm deep learning time series classifier."""
+
+import os
+from copy import deepcopy
+
+import numpy as np
+
+os.environ["CUDA_LAUNCH_BLOCKING"] = "1"
```

Comment:
should this be on module level? This will be called whenever we crawl the package.